scipy/io/matlab/tests/test_miobase.py

```diff
@@ -11,18 +11,19 @@
 
 def test_matdims():
     # Test matdims dimension finder
-    assert_equal(matdims(np.array(1)), (1, 1))  # NumPy scalar
-    assert_equal(matdims(np.array([1])), (1, 1))  # 1-D array, 1 element
-    assert_equal(matdims(np.array([1,2])), (2, 1))  # 1-D array, 2 elements
-    assert_equal(matdims(np.array([[2],[3]])), (2, 1))  # 2-D array, column vector
-    assert_equal(matdims(np.array([[2,3]])), (1, 2))  # 2-D array, row vector
+    assert_equal(matdims(np.array(1)), (1, 1))  # numpy scalar
```

Comment:
Can you undo the changes to comments in this file? The original spellings are the preferred ones, and these comments changes don't serve a function in this PR..